### DISCUSSION TITLE
Stopped download in a little bit

### DISCUSSION THREAD

**Discussion by bohaman1** [aclap-dev/video-downloadhelper#837] [Help and Q&A]
### Your question

Hello,

Before the new update I were able to download the same site fine. However, with the latest version the video link download for about 5 minutes and it said it finished downloading. Anyone have any suggestion?

Thank you 

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"343"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "concurrent_downloads_max": 1,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 22
}

**Comment by rihntintin** [comment]
I'm just another VDH user & not part of the support team.

They need more information, such as: all the pertinent steps you take before, during & after; website name; actual video you're clicking on the site; what happens when you play the downloaded video; examples of the videos you were able to successfully download a video; screenshots; anything you can think of; etc.

Your original question contained absolute nothing regarding your issue. VDH support team needs information to help better understand what you are talking about. Right now, you only said: It's broken. Fix it. lol

I am definitely not making fun or make light of your question. Most users have posted similar questions. I'm just trying to extract more information from you. I hope this helps you.

**Reply by rihntintin** [reply]
Here is a useful link. Please read all of it and follow all the links inside it:
[https://github.com/aclap-dev/video-downloadhelper/discussions/3](url)

Also, read the FAQ and questions from other users, to better understand what the support team needs.

**Reply by GrampaWildWilly** [reply]
I am so encouraged by your stepping in here @rihntintin.  Unfortunately, the link in your post is non-functional.  In any case, the link people should be clicking is the one at the top of every page of this forum labelled Wiki.  The FAQ is there along with instructions for properly opening a problem report.

@bohaman1 You have edited your original post to add the information about your environment.  This is most welcome.  But it's only part of the problem report.  The other part, something @rihntintin pointed out, is still missing: URLs for web pages on which you are having trouble performing your downloads.  The idea is to give enough information for somebody else to replicate your results, or at least try to.  That somebody else might be @rihntintin, who is another user.  Or me, also just another user.  Or one of the developers.  But as things stand at the moment, you haven't supplied enough information for anybody else to act on.  Help yourself.  Give us some URLs exhibiting the problem.
